src/TTextEdit.cpp

```diff
@@ -1655,7 +1655,7 @@ int TTextEdit::bufferScrollUp( int lines )
 
 int TTextEdit::bufferScrollDown( int lines )
 {
-    if( ( mpBuffer->mCursorY + lines ) < (int)(mpBuffer->size()-1 - mScreenHeight) )
+    if( ( mpBuffer->mCursorY + lines ) < (int)(mpBuffer->size()) )
```

Comment:
:bulb: The built-in Qt Creator `clang-analyser` is starting to moan (warn, not error) about "old C-style casts" - it would like:
`if( ( mpBuffer->mCursorY + lines ) < (int)(mpBuffer->size()) )`
to be changed to:
`if( ( mpBuffer->mCursorY + lines ) < static_cast<int>(mpBuffer->size()) )`
and the same sort of thing in three other places I think.

Thoughts anyone?